packages/@aws-cdk/aws-kms/lib/alias.ts

```diff
@@ -160,9 +173,19 @@ export class Alias extends AliasBase {
     if (props.removalPolicy) {
       resource.applyRemovalPolicy(props.removalPolicy);
     }
+
+    this._generatedNameMaxLength = props.generatedNameMaxLength || 256;
   }
 
   protected generatePhysicalName(): string {
```

Comment:
Can you subclass `Alias` and implement this locally in the CodePipeline codebase so this hack doesn't leak into the public API?

Reply:
Done.